x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -249,41 +304,13 @@ public void testDatePartInvalidArgs() {
 
     public void testDatePartValidArgs() {
         accept("SELECT DATE_PART('weekday', date) FROM test");
-        accept("SELECT DATE_PART('dw', date) FROM test");
-        accept("SELECT DATE_PART('tz', date) FROM test");
+        accept("SELECT DATEPART('dw', date) FROM test");
+        accept("SELECT DATEPART('tz', date) FROM test");
         accept("SELECT DATE_PART('dayofyear', date) FROM test");
         accept("SELECT DATE_PART('dy', date) FROM test");
         accept("SELECT DATE_PART('ms', date) FROM test");
     }
 
-    public void testDateAddInvalidArgs() {
```

Comment:
Why make the move?

Reply:
Just to be consistent with the alphabetical ordering of those functions.